src/averageAges.js

```diff
@@ -55,7 +67,17 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const mother = child => people.some(woman => woman.name === child.mother);
```

Comment:
Это функция, название функции должно начинаться с глагола
1. [NAMING] - Name of functions should start with verbs. It will be easier to understand a function's purpose.